docs/sql-programming-guide.md

```diff
@@ -1776,6 +1776,66 @@ working with timestamps in `pandas_udf`s to get the best performance, see
 
 ## Upgrading From Spark SQL 2.2 to 2.3
 
+  - Since Spark 2.3, Spark supports a vectorized ORC reader with a new ORC file format for ORC files and Hive ORC tables. To do that, the following configurations are newly added or change their default values.
```

Comment:
re: `the following configurations are newly added or change their default values.`
these are all new, right?

Reply:
The last two are existing one~